src/Backend/Core/Engine/Authentication.php

```diff
@@ -229,11 +229,10 @@ public static function isAllowedAction($action = null, $module = null)
     {
         $alwaysAllowed = self::getAlwaysAllowed();
 
-        // grab the URL from the reference
-        $URL = BackendModel::get('url');
```

Comment:
why'd you drop this? It's potentially used in 2 places so I'd just leave it there